synapseclient/__main__.py

```diff
@@ -577,6 +578,9 @@ def build_parser():
                             'is deemed fully stable and becomes the default implementation.')
     parser_get.add_argument('id', metavar='syn123', nargs='?', type=str,
                             help='Synapse ID of form syn123 of desired data object.')
+    # add no manifest option
+    parser_get.add_argument('--createManifest', type=str, choices=['all', 'root', 'suppress'],
```

Comment:
The command line option is "createManifest", but the name of the argument to the function is just manifest. Maybe we can make them the same, just "manifest"?